codegen/core/src/main/java/software/amazon/smithy/python/codegen/generators/ConfigGenerator.java

```diff
@@ -324,7 +324,7 @@ private void generateConfig(GenerationContext context, PythonWriter writer) {
         writer.addStdlibImport("dataclasses", "dataclass");
         writer.write("""
                 @dataclass(init=False)
-                class $L:
+                class $T:
```

Comment:
`T` is intended for references, not declarations. Why was this change made?

Reply:
Reverted to $L for the decleration - Had done it without thinking since its using the symbol as an argument.